manifests/03-rbac-role-cluster.yaml

```diff
@@ -3,6 +3,17 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: console-operator
 rules:
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - subjectaccessreviews
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - delete
```

Comment:
I believe only `create` is supported for `subjectaccessreview`

Reply:
good call, updated.